src/Agent/NewRelic/Agent/Core/WireModels/MetricWireModel.cs

```diff
@@ -112,6 +112,14 @@ public override bool Equals(object obj)
             return false;
         }
 
+        public override int GetHashCode()
```

Comment:
What did this do?

Reply:
Spoke with @jifeingo. Theory is that we implemented equality support for the purposes of tests... for use with `Assert.Contains` for example. Apparently the support was not fully implemented. Adding the `GetHashCode` method resolved a build warning.